tests/test_clients.py

```diff
@@ -1227,21 +1333,21 @@ def test_load_modules_with_python_module_path_respects_all(self):
 
             import_module.assert_called_once_with("okokok.no.u")
 
-        mock_module.loader.assert_called_once_with(client)
-        mock_module.other_loader.assert_not_called()
-        priv_loader.assert_called_once_with(client)
-        mock_module.another_loader.assert_called_once_with(client)
+        mock_module.loader.load.assert_called_once_with(client)
+        mock_module.other_loader.load.assert_not_called()
```

Comment:
this naming scheme with the loaders and unloaders in these tests is inconsistent and should be revised